### PR TITLE
Modify location repeater response for BETS

### PR DESCRIPTION
**Pull request by esoergel**
@dannyroberts @nickpell

### CODE REVIEW

File: corehq/apps/repeaters/repeater_generators.py [hunk 1]
```diff
@@ -315,8 +315,6 @@ def get_test_payload(self, domain):
 
 class UserPayloadGenerator(BasePayloadGenerator):
 
-    deprecated_format_names = ('user_json',)
```
Comment (esoergel) [inline corehq/apps/repeaters/repeater_generators.py]: @dannyroberts I checked for these on prod and india and didn't find anything.  The `all_docs/by_doc_type` view isn't synced to the receiverwrapper db, unfortunately, so I had to go domain-by-domain.

Comment (dannyroberts) [inline corehq/apps/repeaters/repeater_generators.py]: oof, dedication

Comment (dannyroberts) [inline corehq/apps/repeaters/repeater_generators.py]: Just checking but I'm assuming what you said about `LocationPayloadGenerator` also applies to `UserPayloadGenerator`? I.e. those repeaters were both only recently created and have no live usages.

**Review by dannyroberts — APPROVED** [review]

File: corehq/apps/repeaters/repeater_generators.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/repeaters/repeater_generators.py]: correct, I checked both

Comment (dannyroberts) [inline corehq/apps/repeaters/repeater_generators.py]: 👍 thanks

**Pull request merged**